### ISSUE TITLE
Training code BUG?

### ISSUE THREAD

**Issue by songzhaozhe** [facebookresearch/darkforestGo#31]
In line 311 in darkforestGo/train/rl_framework/infra/bundle.lua: 
self.params[i]:add(-learning_rate, self.gparams[i])

I think this is weird since adding the learning rate with the gradient makes no sense,  Maybe the author wanted to use 'addcmul' instead of 'add'?

**Comment by yuandong-tian** [comment]
Please check this document: 

https://github.com/torch/torch7/blob/master/doc/maths.md

```
x:add(value, y) multiply-accumulates values of y into x.
```

**Comment by songzhaozhe** [comment]
@yuandong-tian  Thank you! Sorry for the silly question. I missed out that sentence when reading the docs.
